tests/models/gemma/test_modeling_gemma.py

```diff
@@ -531,12 +499,21 @@ def test_flash_attn_2_equivalence(self):
                 assert torch.allclose(logits_fa, logits, atol=3e-3)
 
 
-@require_torch_gpu
 @slow
-@require_read_token
+@require_torch_gpu
 class GemmaIntegrationTest(unittest.TestCase):
     input_text = ["Hello I am doing", "Hi today"]
+    # This variable is used to determine which CUDA device are we using for our runners (A10 or T4)
+    # Depending on the hardware we get different logits / generations
+    cuda_compute_capability_major_version = None
 
+    @classmethod
+    def setUpClass(cls):
+        if is_torch_available() and torch.cuda.is_available():
+            # 8 is for A100 / A10 and 7 for T4
+            cls.cuda_compute_capability_major_version = torch.cuda.get_device_capability()[0]
+
+    @require_read_token
```

Comment:
This should be a better name, cc @younesbelkada 

See https://github.com/huggingface/transformers/pull/29805#discussion_r1535373326